.github/workflows/build-mariadb.yml

```diff
@@ -118,12 +118,9 @@ jobs:
           mv froxlor-nightly.${{steps.vars.outputs.sha_short}}.zip dist/
           mv froxlor-nightly.${{steps.vars.outputs.sha_short}}.zip.sha256 dist/
 
-      - name: Deploy nightly to server
```

Comment:
please dont change stuff in the build - this needs to be available on our servers for the update/autoupdate to work and does not have anything to do with the feature you are working on in any way